spaces/ETG.Finance/index.json

```
@@ -0,0 +1,29 @@
+{
+  "token": "0x74603e780545d02C4257E7D2BE19c74dE7BE1952", 
+  "name": "ETG.Finance", 
+  "network": "1", 
+  "symbol": "ETGF", 
+  "skin": "etgfinance", 
+  "domain": "governance.etgproject.org", 
+  "strategies": [ 
+    {
+      "name": "erc20-balance-of", 
+      "params": { 
+        "address": "0x74603e780545d02C4257E7D2BE19c74dE7BE1952", 
+        "symbol": "ETGF", 
+        "decimals": 18 
+      }
+    }
+  ],
+  "members": [ 
+    "0xf14e2952a6a71c342cba3d7aa4ed932151964945" 
+  ],
+  "filters": { 
+    "defaultTab": "all", 
+    "minScore": 0.01, 
+    "onlyMembers": true 
+  },
+  "invalids": [ 
+    "0x6a36222e45265640B636e6BD2b558d1fB81458E6" 
+  ]

```

The `invalids` field should be move in `filters`.
```suggestion
  "filters": { 
    "defaultTab": "all", 
    "minScore": 0.01, 
    "onlyMembers": true,
    "invalids": [ 
      "0x6a36222e45265640B636e6BD2b558d1fB81458E6" 
    ]
  }
```